rasa/shared/utils/validation.py

```diff
@@ -247,6 +247,18 @@ def validate_training_data_format_version(
             raise TypeError
 
         if version.parse(LATEST_TRAINING_DATA_FORMAT_VERSION) >= parsed_version:
+            if version_value < LATEST_TRAINING_DATA_FORMAT_VERSION:
```

Comment:
I think this should be a standalone if statement, not a nested one, so should be placed just beneath this one and return `False`.

Reply:
If it returns False, what then happens? Won't the validation fail, and the training not run?

Reply:
Shouldn't it simply not return anything? If we return False, won't the validation fail and the training not run?

Reply:
Yes, that is what happens:
<img width="938" alt="Screenshot 2021-11-22 at 15 08 37" src="https://user-images.githubusercontent.com/1754975/142875916-6164efac-0029-4d03-9254-32cf4ada5461.png">

I'll just have it return nothing.
This is what the output now looks like:
<img width="936" alt="Screenshot 2021-11-22 at 15 10 03" src="https://user-images.githubusercontent.com/1754975/142876104-fad2e65d-27d7-47b8-9706-fd740c6b87d1.png">

<img width="940" alt="Screenshot 2021-11-22 at 15 10 45" src="https://user-images.githubusercontent.com/1754975/142876219-8f3fc7a2-0072-47f7-a34b-1c0920c633c5.png">